action.yaml

```diff
@@ -103,6 +103,8 @@ runs:
       id: exclude
```

Comment:
FYI -- I put this bash code into a file and then ran shellcheck on it. There were a number of violations like `SC2296 (error): Parameter expansions can't start with {. Double check syntax.` which I didn't attempt to fix here.

Reply:
That's expected, it's not bash, they are for the GitHub action parser.